.github/workflows/ci.yml

```diff
@@ -204,16 +204,12 @@ jobs:
         - cp3{9,10,11,12}-macosx_arm64
     secrets:
       pypi_token: ${{ secrets.pypi_token }}
-      anaconda_token: ${{ secrets.anaconda_token }}
+      anaconda_token: ${{ secrets.anaconda_org_upload_token }}
 
   publish_pure:
     needs: [publish]
     uses: OpenAstronomy/github-actions-workflows/.github/workflows/publish_pure_python.yml@main
     with:
-      upload_to_anaconda: ${{ (github.event_name == 'schedule' || github.event_name == 'workflow_dispatch') }}
```

Comment:
Anaconda uploads are for wheels only.

Reply:
This should be the universal wheel? maybe it's also the sdist.

Reply:
I think the issue is that if we don't also upload the universal wheel if you try to use the nightly pip index on windows it will explode (as it won't be able to build from source).

Reply:
It uploaded both the pure wheel and the dist. Maybe if we can get the OA template to only upload the wheels?